roles/repository_sync/tasks/main.yml

```
@@ -0,0 +1,35 @@
+---
+
+# Sync AH repository
+- name: Sync Automation Hub repository
+  ah_repository_sync:
+    name:                       "{{ __repository_item.name }}"
+    wait:                        "{{ __repository_item.wait | default(omit)  }}"
+    auth_url:                   "{{ __repository_item.interval | default(1) }}"
+    timeout:                      "{{ __repository_item.timeout | default(omit) }}"
+  loop: "{{ ah_repositories }}"
+  loop_control:
+    loop_var: "__repository_item"
+  no_log: "{{ ah_configuration_repository_secure_logging }}"
+  async: 1000
+  poll: 0
+  register: __repositories_syncs_job_async
+  changed_when: not __repositories_syncs_job_async.changed
+  vars:
+    ansible_async_dir: '/tmp/.ansible_async'
+
+- name: "Repository Sync | Wait for finish the repository_sync to finish"
+  async_status:
+    jid: "{{ __repositories_syncs_job_async_result_item.ansible_job_id }}"
+  register: __repositories_syncs_job_async_result
+  until: __repositories_syncs_job_async_result.finished
+  retries: "{{ ah_configuration_repository_syncs_async_retries }}"
+  delay: "{{ ah_configuration_repository_syncs_async_delay }}"
+  loop: "{{ __repositories_syncs_job_async.results }}"
+  loop_control:
+    loop_var: __repositories_syncs_job_async_result_item
+  when: __repositories_syncs_job_async_result_item.ansible_job_id is defined
+  no_log: "{{ ah_configuration_repository_syncs_secure_logging }}"
```

See what you mean. This one needs to change to `ah_configuration_repository_secure_logging`

```suggestion
  no_log: "{{ ah_configuration_repository_secure_logging }}"
```

---

No, ah_configuration_repository_secure_logging is already used in a role which is why I was trying to say I think it should be ah_configuration_repository_syncs_secure_logging instead

---

Exactly, it should be the same var for the `repository`, `repository_index` and `repository_sync` role

---

See that ee_registry_X uses the same idea:
https://github.com/redhat-cop/ah_configuration/blob/e1dfbab02ef95afd54f2d03c7c4bcb60988129f9/roles/ee_registry_index/tasks/main.yml#L18

---

Updated with the `ah_configuration_repository_secure_logging` as suggested